README.md

```diff
@@ -282,6 +282,26 @@ When `user1` logs in, he/she will be able to manage `myAppId1` and `myAppId2` fr
 
 When *`user2`*  logs in, he/she will only be able to manage *`myAppId1`* from the dashboard.
 
+## Configuring Localized Push Notifications
+
+With the latest version of the dashboard, it is possible to send localized messages for push notifications.
```

Comment:
make dashboard a link.  for bonus points make push notifications a link to doc too ;)